src/Config/SaferPayConfig.php

```diff
@@ -88,7 +88,7 @@ class SaferPayConfig
     const PAYMENT_INVOICE = 'INVOICE';
     const PAYMENT_JCB = 'JCB';
     const PAYMENT_MAESTRO = 'MAESTRO';
-    const PAYMENT_MASTERCARD = 'MasterCard';
+    const PAYMENT_MASTERCARD = 'MASTERCARD';
```

Comment:
Damn was that change added by me not so long ago? If so then i remember there was issue that the name was not like 'MasterCard' :D maybe i m wrong but if its so then i think there might be issue that sometimes the mastercard name is different. I hope i m wrong and this will fix it.

Reply:
I got an answer from the integration team that MasterCard must be uppercase. They also admitted that in the backend they had issue witch sometimes failed validation in lowercase and in some cases it did not. We and them took measures to fix this.

Reply:
https://invertus.atlassian.net/jira/software/c/projects/SL/boards/268?modal=detail&selectedIssue=SL-117
In this ticket is more information from their answer